substrate/frame/session/benchmarking/src/mock.rs

```diff
@@ -79,7 +80,8 @@ sp_runtime::impl_opaque_keys! {
 
 pub struct TestSessionHandler;
 impl pallet_session::SessionHandler<AccountId> for TestSessionHandler {
-	const KEY_TYPE_IDS: &'static [sp_runtime::KeyTypeId] = &[];
+	// corresponds to the opaque key id above
+	const KEY_TYPE_IDS: &'static [KeyTypeId] = &[KeyTypeId([100u8, 117u8, 109u8, 121u8])];
```

Comment:
This panicked in for `cargo test -p pallet-session-benchmarking --features runtime-benchmarks`, but it also failed on master. Is this not CI tested?

```console

---- mock::test_genesis_config_builds stdout ----
thread 'mock::test_genesis_config_builds' panicked at /home/clang/code/polkadot-sdk/substrate/frame/session/src/lib.rs:440:17:
Number of keys in session handler and session keys does not match
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace

```

Reply:
I opened an issue: https://github.com/paritytech/polkadot-sdk/issues/6726